deployment/docker-compose.yml

```diff
@@ -7,18 +7,21 @@ services:
     ports:
       - 80:80
     volumes:
-      - ./nginx.conf:/etc/nginx/conf.d/default.conf
+      - $HOME/deployment/nginx.conf:/etc/nginx/conf.d/default.conf
```

Comment:
why cant we keep it as its been fore? because of ubuntu user? I would rather keep the yaml as simple as possible